code-challenges.test.js

```diff
@@ -51,14 +91,32 @@ const studyMinutesWeek2 = {
 }
 // Expected output: [10, 15, 20, 45, 60, 65, 100]
 
-// b) Create the function that makes the test pass.
+
+
+// // b) Create the function that makes the test pass.
+
+const dayArray = (object) => {
+  return Object.values(object).sort((a, b) => (a - b) )
+}
+// console.log(dayArray(studyMinutesWeek1))
+// console.log(dayArray(studyMinutesWeek2))
 
 // Pseudo code:
 
+// 1.) Declare a function that takes the value of the object and displays the value in the array. 
```

Comment:
Definitely need more explanation here, you basically only have your input and output. You should include that you are using Object.values to return an array of the values of the object, then using sort to sort from least to greatest.